src/erlcloud_sns.erl

```diff
@@ -224,11 +226,11 @@ delete_topic(TopicArn, Config)
 
 
 
--spec get_endpoint_attributes/1 :: (string()) -> sns_endpoint().
+-spec get_endpoint_attributes/1 :: (parameter()) -> sns_endpoint().
 get_endpoint_attributes(EndpointArn) ->
     get_endpoint_attributes(EndpointArn, default_config()).
 
--spec get_endpoint_attributes/2 :: (string(), aws_config()) -> sns_endpoint().
+-spec get_endpoint_attributes/2 :: (parameter(), aws_config()) -> sns_endpoint().
```

Comment:
Line 233 is too long: -spec get_endpoint_attributes/2 :: (parameter(), aws_config()) -> sns_endpoint()..